corehq/apps/domain/templates/domain/internal_subscription_management.html

```diff
@@ -0,0 +1,66 @@
+{% extends 'hqwebapp/base_section.html' %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% load hq_shared_tags %}
+
+{% block js %}{{ block.super }}
+    <script src="{% static 'hqwebapp/js/lib/jquery-ui-datepicker/jquery-ui-1.8.17.custom.datepicker.min.js' %}"></script>
+    <script src="{% static 'hqwebapp/js/lib/select2/select2.js' %}"></script>
+    <script src="{% static 'accounting/js/accounting.billing_info_handler.js' %}"></script>
+{% endblock %}
+
+{% block js-inline %}{{ block.super }}
+    <script type="text/javascript">
+        var viewModel = {
+            subscriptionType: ko.observable($('#id_subscription_type').val() || null)
+        };
+        ko.applyBindings(viewModel, document.getElementById('subscription_management'));
+    </script>
+
+    <script>
```

Comment:
there are a lot of open script close script tags. maybe we can just make it one?